ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml

```
@@ -21,6 +21,14 @@
     - ansible_distribution == "Solaris"
   tags: [jenkins_user, adoptopenjdk]
 
+- name: Create Jenkins  account group for SLES15/s390x
```

Erroneous space character
```suggestion
- name: Create Jenkins account group for SLES15/s390x
```